data/input_2022/TD/intel-wot-ha/TDs/dir2.td.jsonld

```diff
@@ -0,0 +1,178 @@
+{
+    "@context": [
+        "https://www.w3.org/2022/wot/td/v1.1",
+        "https://w3c.github.io/wot-discovery/context/discovery-context.jsonld",
+        { "@language": "en" }
+    ],
+    "@type": [
+        "ThingDirectory"
+    ],
+    "title": "Basic Thing Description (TD) Directory (TDD)",
+    "titles": {
+        "en": "Basic W3C WoT Thing Description (TD) Directory (TDD)",
```

Comment:
This title is different from the title above

Reply:
Ah, you're right.  Ok, let me correct and retry the validation.